src/views/Vacancies/components/FilterTagsHeader.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Chip, Typography } from "@material-ui/core";
+import _get from "lodash/get";
+
+const useStyles = makeStyles(theme => ({
```

Comment:
in the media channels browser page, @E1379 will also have to use filter tags. how can we reuse rather than have separate implementations for each?

Reply:
I think Emil has added different widget which displays autocomplete tags.